packages/serverless/src/model/database/group.ts

```diff
@@ -6,7 +6,7 @@ import {
   ModelGeneral,
   zkDatabaseConstant,
 } from '@zkdb/storage';
-import { ClientSession, InsertOneOptions, WithoutId } from 'mongodb';
+import { ClientSession, WithoutId } from 'mongodb';
 
 export class ModelGroup extends ModelGeneral<WithoutId<TGroupRecord>> {
```

Comment:
I think `OptionalId` is better than `WithoutId`, I was wrong.

Reply:
I agree, change all group user and database model to `OptionalId` instead of `WithoutId`